sdks/go/test/load/util.go

```diff
@@ -132,10 +133,16 @@ func newLoadTestResult(value float64) loadTestResult {
 // PublishMetrics calculates the runtime and sends the result to InfluxDB database.
 func PublishMetrics(results metrics.QueryResults) {
 	options := newInfluxDBOptions()
-	if options.validate() {
-		if res := toLoadTestResults(results); len(res) > 0 {
-			publishMetricstoInfluxDB(options, toLoadTestResults(results))
-		}
+	ress := toLoadTestResults(results)
+	for _, res := range ress {
+		log.Printf("%s %v", res.metric, time.Duration(float64(time.Second)*res.value))
+	}
+	if len(ress) == 0 {
+		log.Print("No metrics returned.")
+		return
+	}
+	if options.validate() && len(ress) > 0 {
```

Comment:
```suggestion
	if options.validate() {
```

We've already early returned in this case so we probably don't need this. If you want to be extra explicit you can put it in the else (I'd just cut the extra clause though)

Reply:
Nice catch. Nah, no metrics means it doesn't mean anything for the DB options to be correct or not.